po/it.po

```diff
@@ -458,42 +450,39 @@ msgstr "Incolla inserendo pagine _pari"
 
 #: data/menu.ui.h:27
 msgid "Insert Blan_k Page"
-msgstr ""
+msgstr "Inserisci pagina vuota"
 
 #: data/menu.ui.h:28
 msgid "_Select"
-msgstr ""
+msgstr "_Seleziona"
 
 #: data/menu.ui.h:29
 msgid "Select _All"
-msgstr ""
+msgstr "_Seleziona_tutto"
 
 #: data/menu.ui.h:30
 msgid "_Deselect All"
-msgstr ""
+msgstr "_Deseleziona Tutto"
 
 #: data/menu.ui.h:31
-#, fuzzy
 msgid "Select _Odd Pages"
-msgstr "Taglia pagine selezionate"
+msgstr "Seleziona_Pagine dispari"
 
 #: data/menu.ui.h:32
-#, fuzzy
 msgid "Select _Even Pages"
-msgstr "Taglia pagine selezionate"
+msgstr "Seleziona_Pagine pari"
 
 #: data/menu.ui.h:33
 msgid "All From _Same File"
-msgstr ""
+msgstr "Tutto dallo_stesso file"
```

Comment:
Is it dallostesso or dallo stesso? In case of the latter a space is missing. (See above.)